meetballs/settings.py

```diff
@@ -40,7 +40,9 @@
     'meet_balls_app.apps.MeetBallsAppConfig',
     'player_rating.apps.PlayerRatingConfig',
     'court.apps.CourtConfig',
-    'player.apps.PlayerConfig'
+    'player.apps.PlayerConfig',
+    'game_event.apps.GameEventConfig',
+    'game_event_player.apps.GameEventPlayerConfig'
 ]
```

Comment:
this PR, doesn't need line 45 here, so if you remove it it would solve your test fail, am i wrong ?

if i am right, that would make your PR ez to merge.

Reply:
> this PR, doesn't need line 45 here, so if you remove it it would solve your test fail, am i wrong ?
> 
> if i am right, that would make your PR ez to merge.

Amazing! you're right, it fixes the problem
Thank you